<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="admin-PC" name="CreateClient" tests="3" failures="1" timestamp="2 May 2014 13:43:12 GMT" time="24.377" errors="0">
  <testcase name="L5_Create_client_Validations" time="7.611" classname="scenarios.CreateClient"/>
  <testcase name="L5_Create_client" time="16.604" classname="scenarios.CreateClient">
    <failure type="java.lang.AssertionError" message="Validate that success message is displayed. expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Validate that success message is displayed. expected [true] but found [false]
	at framework.GeneralMethods.Textvalidation(GeneralMethods.java:287)
	at framework.GeneralMethods.action(GeneralMethods.java:63)
	at framework.Standard.StandardM1(Standard.java:21)
	at scenarios.CreateClient.L5_Create_client(CreateClient.java:55)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- L5_Create_client -->
  <testcase name="L5_ResetPassword" time="0.001" classname="scenarios.CreateClient">
    <skipped/>
  </testcase> <!-- L5_ResetPassword -->
</testsuite> <!-- CreateClient -->
